### PR TITLE
redesign channel page

### PR DESCRIPTION
**Pull request by seanyesmunt**
### Changes
- New channel page design
  - Created `<ChannelThumbnail />`,  `<ChannelContent />`,  and `<ChannelAbout />`
- New `<Paginate />` component since we had that logic in multiple places
- Cleaned up some un-needed styles related to the `<main>` styling in the `<Page />` component.

#### Notes
The "edit" mode should be pretty straight forward after this. Will probably need to break out the cover photo component, but everything is pretty isolated.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -22,20 +22,22 @@
...
-    "comma-dangle": ["error", "always-multiline"],
+    "object-curly-spacing": 0,
+    "one-var": 0,
+    "prefer-promise-reject-errors": 0,
+    "react-hooks/exhaustive-deps": "warn",
+    "react-hooks/rules-of-hooks": "error",
     "space-before-function-paren": ["error", "never"],
-    "jsx-quotes": ["error", "prefer-double"],
+    "standard/object-curly-even-spacing": 0,
+    "standard/no-callback-literal": 0,
```
Comment (seanyesmunt) [inline .eslintrc.json]: Making these ^ in alphabetical order

File: src/ui/component/router/view.jsx [hunk 2]
```diff
@@ -58,8 +58,11 @@ export default function AppRouter() {
         <Route path={`/$/${PAGES.HISTORY}/all`} exact component={NavigationHistory} />
 
         {/* Below need to go at the end to make sure we don't match any of our pages first */}
-        <Route path="/:claimName/:claimId" component={ShowPage} />
-        <Route path="/:claimName" component={ShowPage} />
+        <Route path="/:claimName" exact component={ShowPage} />
+        <Route path="/:claimName/:claimId" exact component={ShowPage} />
```
Comment (kauffj) [inline src/ui/component/router/view.jsx]: https://github.com/lbryio/lbry-desktop/issues/2463

File: src/ui/scss/component/_main.scss [hunk 3]
```diff
@@ -35,7 +49,7 @@
     'content content'
     'info    related';
 
-  @media (min-width: 1470px) {
+  @media (min-width: 1300px) {
```
Comment (kauffj) [inline src/ui/scss/component/_main.scss]: should this be a constant? is this value used elsewhere?

File: src/ui/scss/component/_search.scss [hunk 4]
```diff
@@ -53,7 +41,7 @@
 
 .search__options-wrapper {
   font-size: 1.25em;
-  padding-bottom: var(--spacing-vertical-large);
+  margin-bottom: var(--spacing-vertical-large);
```
Comment (kauffj) [inline src/ui/scss/component/_search.scss]: Please continue to do this. @NetOperatorWibby is on the same page that we should typically be using margin over padding without a strong reason to prefer padding.

File: src/ui/scss/component/tabs.scss [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    margin-left: auto;
+  }
+
+  [data-mode='dark'] & {
+    background-color: darken($lbry-black, 5%);
+  }
+}
+
+.tabs__list--channel-page {
+  padding-left: calc(10rem + var(--spacing-vertical-large) + var(--spacing-vertical-xlarge));
```
Comment (kauffj) [inline src/ui/scss/component/tabs.scss]: What is special about channel tabs?

Comment (seanyesmunt) [reply src/ui/scss/component/tabs.scss]: Channel tabs are slid to the right because they have to deal with the channel thumbnail photo overlap from above.

Comment (kauffj) [reply src/ui/scss/component/tabs.scss]: :+1:

File: src/ui/scss/component/_main.scss (hunk 3 above)
Comment (seanyesmunt) [reply src/ui/scss/component/_main.scss]: Maybe. We aren't using it anywhere else. I just wanted to lower it so I can see the related content to the right on my laptop.

File: src/ui/component/channelAbout/view.jsx [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+function ChannelContent(props: Props) {
+  const { description, email, website } = props;
+  const showAbout = description || email || website;
+
+  return (
+    <section>
+      {!showAbout && <h2 className="empty">{__('Nothing here yet')}</h2>}
+      {showAbout && (
+        <Fragment>
+          {description && <div className="channel__description">{description}</div>}
```
[the remark this answers is not in the document]
Comment (seanyesmunt) [reply src/ui/component/channelAbout/view.jsx]: Yeah just made it the same as the other "items". Will probably change it a little once I see what more youtuber's descriptions are.

**Comment by seanyesmunt** [comment]
@kauffj I pushed up a new commit. I guess I must have removed some eslint rules at some point. There were a number of unused imports that are now removed.

**Pull request merged**